package.json

```diff
@@ -31,7 +31,8 @@
   "pnpm": {
     "overrides": {
       "jsdom@^20.0.3>nwsapi@^2.2.0": "2.2.9",
-      "vite>@types/node": "^20.17.16"
+      "vite>@types/node": "^20.17.16",
+      "electron-vite@^2.3.0>esbuild@0.21.5": "^0.25.0"
```

Comment:
According to the changelog, there are three releases with breaking changes:

* https://github.com/evanw/esbuild/releases/tag/v0.25.0
* https://github.com/evanw/esbuild/releases/tag/v0.24.0
* https://github.com/evanw/esbuild/releases/tag/v0.23.0

The next batch of dependabot updates that goes out next week is going to update electron-vite anyway, which means we'd try to perform a tag build and do some light testing of Connect. Would it make sense to wait for the update and force the esbuild in the PR that updates electron-vite? We'd be then going through only a single update with breaking changes (esbuild 0.24.2 required by electron-vite 3.0.0 to esbuild 0.25.0) instead of three.

Linking this here for tracking:

* https://github.com/alex8088/electron-vite/issues/722

Reply:
sure, dropping this then